service/src/main/java/org/jboss/sbomer/model/Sbom.java

```diff
@@ -116,16 +117,22 @@ public class Sbom extends PanacheEntityBase {
     private String buildId;
 
     @Column(name = "root_purl")
-    @NotBlank(message = "Root purl identifier missing")
+    // @NotBlank(message = "Root purl identifier missing") // TODO
```

Comment:
So, I had to make it optional. Currently I change the approach and the SBOMs are saved by the service while not being fully populated with the actual Bom data. This is to support statuses of Sbom  resources, for examle: we don't have the sbom, because it's still generating and the status is `GENERATING`. And because of this, the purl will be empty at that stage.